docs/en/Security.md

```diff
@@ -9,71 +9,48 @@ priority: 1
-the user accessing it. It is the basis for other security features such as authorization.
-2. [Authorization](#authorization): If enabled, Alluxio file system can control the user's access.
-POSIX permission model is used in Alluxio to assign permissions and
-control access rights.
-3. [Auditing](#auditing): If enabled, Alluxio file system can maintain an audit log for users' accesses
-to file metadata.
-
-By default Alluxio runs in SIMPLE secure mode in which a simple authentication is required.
-SIMPLE indicates that server trusts whoever the client claims to be.
-See [Security specific configuration](Configuration-Settings.html#security-configuration) to
-enable and use security features.
+This document describes the following security related features in Alluxio.
+
+1. [Authentication](#authentication): If `alluxio.security.authentication.type=SIMPLE`,
+Alluxio file system recognizes the user accessing the service.
+Having `SIMPLE` authentication is required to use other security features such as authorization.
+Alluxio also supports other authentication modes like `NOSASL` and `CUSTOM`.
+2. [Authorization](#authorization): If `alluxio.security.authorization.permission.enabled=true`,
+Alluxio file system will grant or deny user access based on the requesting user and the POSIX
+permission model of the files or directories to access.
+Note that, authentication cannot be `NOSASL` as authorization requires user information.
+3. [Auditing](#auditing): If `alluxio.master.audit.logging.enabled=true`, Alluxio file system can
+maintain an audit log for user accesses to file metadata.
+
+See [Security specific configuration](Configuration-Settings.html#security-configuration) for
+different security properties.
 
 ## Authentication
 
-Alluxio provides file system service through Thrift RPC. The client side (representing a user)
-and the server side (such as master) should build an authenticated connection for communication.
-If authentication succeeds, the connection will be built. If fails,
-the connection can not be built and an exception will be thrown to client.
-
-Three authentication modes are supported: NOSASL, SIMPLE (default mode), and CUSTOM.
-
-### User Accounts
-
-The communication entities in Alluxio consist of master, worker, and client. Each of them needs
-to know its user who are running it, also called as the login user. JAAS (Java Authentication and
-Authorization Service) is used to determine who is currently executing the process.
-
-When authentication is enabled, a login user for the component (master, worker, or client)
-can be obtained by following steps:
+### SIMPLE
 
-1. Login by configurable user. If property 'alluxio.security.login.username' is set by
-application, its value will be the login user.
-2. If its value is empty, login by OS account.
+When `alluxio.security.authentication.type=SIMPLE`, authentication is enabled.
+Before an Alluxio client accessing the service, this client retrieve the user information to report
```

Comment:
"accessing" => "accesses"
"this client retrieve" => "the client retrieves"

Reply:
fixed